content/acra/acra-in-depth/data-flow/_index.md

```diff
@@ -180,103 +180,6 @@ Application requests data from a database through AcraServer. AcraServer transpa
 
 ---
 
-### Using AcraConnector and AcraServer
```

Comment:
let's not remove the docs, because AcraConnector is deprecated but still could be used by old versions. Lets' add deprecation warnings here